Sun.java

```diff
@@ -0,0 +1,12 @@
+public class Sun {
+
+    static int count;
+    public static final String OUTPUT_TEXT = "Солнце вставало следующее количество раз: ";
+
+    public static void daybreak (int count) {
+        System.out.println(OUTPUT_TEXT + count);
```

Comment:
Let's increase counter when calling daybreak method instead of using as parameter, I mean:
public static void daybreak () {
        count+=1;
}
public static void sunrisesCount() {
      System.out.println(OUTPUT_TEXT + count)
}